src/main/java/xyz/niflheim/stockfish/engine/UCIEngine.java

```diff
@@ -92,7 +105,22 @@ private void passOption(Option option) {
     }
 
     private String getPath(Variant variant, String override) {
-        StringBuilder path = new StringBuilder(override == null ? "assets/engines/stockfish_10_x64" : override + "stockfish_10_x64");
+        return getPath(variant, override, null);
+    }
+
+    private String getPath(Variant variant, String override, Integer engineVersion) {
```

Comment:
Regarding this method, I wrote in a comment about the Util class.

Reply:
Addressed